tests/src/readytorun/mainv2.csproj

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I understand that it is not easy to fix the test harness. I think it is good enough for now.